docs/ca/dev/preparation-for-going-live.md

```diff
@@ -23,191 +23,108 @@ We've divided the preparation into approximate timeframes, and you can adjust th
+| APP-16 | Application                    | Implement general security guidelines                        | Apply [security guidelines](/docs/dg/dev/guidelines/security-guidelines.html).                                                                                                                                                                                                                                                                       |
+| APP-17 | Application                    | Implement password security guidelines                       | Make sure that you don't have any plain-text passwords, private keys, or API secrets in config files or Git repositories.                                                                                                                                                                                                                                    
+| APP-18 | Application                    | Implement credentials security guidelines                    | Minimize the use of personal credentials and choose work-specific accounts based on each environment: production, staging, testing. We highly recommend employing Centralized Credential Management to securely store and manage these credentials.                                                                                                         
+| APP-19 | Application                    | Implement secrets rotations                                  | Secrets, like API tokens, should be rotated regularly. Outline and test rotation strategies to prevent issues during live operation.                                                                                                                                                                                            
+| APP-20 | Application                    | Implement internal update security guidelines                | Install all the [security updates](/docs/about/all/releases/product-and-code-releases.html) from all Spryker packages.                                                                                                                                                                                                                                       
+| APP-21 | Application                    | Implement external update security guidelines                | Install all the security updates from all the external packages.                                                                                                                                                                                                                                                                                             | To check if your project modules require security updates, you can use the [security checker](/docs/dg/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/spryker-security-checker.html).                                                                 
```

Comment:
How is the checker of Spryker modules related to external security guidelines?

Reply:
Checker has that check, for external security packages (specifically if there is a known vulnerability in your version of external package )

Reply:
If the project needs to update external packages, it should be called `Install external security updates`. What do you think?

Reply:
Perfect!